docs/toolbox/accessibility_indicators/heatmaps/gravity.md

```diff
@@ -237,7 +238,7 @@ Depending on your configuration, the calculation might take a few minutes. The [
 
 TODO: Update result image
 
-![Catchment Area Calculation Result in GOAT](/img/toolbox/accessibility_indicators/catchments/catchment_result.png "Catchment Area Calculation Result in GOAT")
+![Heatmap Gravity-Based Calculation Result in GOAT](/img/toolbox/accessibility_indicators/heatmaps/gravity_based/heatmap_gravity_result.webp "Heatmap Gravity-Based Calculation Result in GOAT")
```

Comment:
This image looks nice!
Small change: could we avoid highlighting the accessibility value? This value is not easy to understand and so we would rather encourage users to observe the color gradient and visual aspects of the result.